MAINTAINERS.yml

```diff
@@ -327,8 +327,10 @@ CMSIS-DSP integration:
     - stephanosio
   collaborators:
     - galak
+    - yperess
   files:
     - modules/Kconfig.cmsis_dsp
+    - subsys/dsp/
```

Comment:
There should be a separate area for "DSP subsystem" since, although the API is identical to that of the CMSIS-DSP, it is supposed a standalone math/DSP subsystem for Zephyr.

Reply:
Done